MoviesApp/Presentation/Scenes/MoviesList/MoviesListViewModel.swift

```diff
@@ -14,7 +15,9 @@ final class MoviesListViewModel {
     func fetchMovies(pagina: Int) {
         guard pagina <= lastPage else { return }
         isLoading.value = true
+        isFetchingNextPage = true
         getMoviesUseCase.execute(pagina: pagina, viewModelCompletionHandler: { [weak self] movies, error in
+            isFetchingNextPage = false
```

Comment:
Are your sure this doesn't make a memory leak????
```suggestion
            self?.isFetchingNextPage = false
```

Reply:
> 

Thank you for catching that. Fixing it!